.travis.yml

```diff
@@ -23,7 +23,8 @@ install:
       pip install --no-deps -e .
       cd ../
     - python setup.py install
-    - npm install -g https://github.com/INCF/bids-validator.git#0.26.0
+    # - npm install -g https://github.com/INCF/bids-validator.git#0.26.0
+    - npm install -g https://github.com/robertoostenveld/bids-validator.git#eeg-validator  # noqa use the EEG fork until merge
```

Comment:
This is the EEG fork for the bids-validator that we are working on. It's also applied for the BEP_006 branch in BIDS-examples, see [here](https://github.com/INCF/BIDS-examples/blob/bep006_eeg/.travis.yml) ... for MEG etc. it is the same

Reply:
okay great !